backend/src/schema/resolvers/follow.spec.js

```diff
@@ -129,6 +129,16 @@ describe('follow', () => {
         data: { followUser: expectedUser },
         errors: undefined,
       })
+
+      // Test to make sure FOLLOWS relationship has "createdAt" date.
+      const relation = await neode.cypher(
+        'MATCH (user:User {id: {id}})-[relationship:FOLLOWS]->(followed:User) WHERE relationship.createdAt IS NOT NULL RETURN relationship',
+        { id: 'u1' },
+      )
+      const relationshipProperties = relation.records.map(
+        record => record.get('relationship').properties.createdAt,
+      )
+      expect(relationshipProperties[0]).toEqual(expect.any(String))
```

Comment:
can you please place this into it's own `it` block? 
we should avoid multiple `expect` in the same it block

Reply:
you can run `await mutate({ mutation: mutationFollowUser, variables })` then this code

Reply:
@mattwr18 -  Much appreciated your review. Implemented changes as you requested.